KD_Lib/common/base_class.py

```diff
@@ -4,7 +4,7 @@
 import torch.optim as optim
 from torch.utils.tensorboard import SummaryWriter
 
-import matplotlib.pyplot as plt 
+import matplotlib.pyplot as plt
```

Comment:
Okay these type of errors should have been taken care. Can you update the travis test to use black? and run it for all files. So the next time we don't have to do this.

Reply:
I'll update the contributing guidelines.

Reply:
I do not know much how travis config works, will read and send a PR, if it works for you.

Reply:
you are supposed to add a single line in the script part in .travis.yml `black --check KD_Lib`. This is sufficient imo.

Reply:
oh, nice! will add :+1: